packages/create-app/templates/default-app/packages/app/src/App.tsx

```diff
@@ -40,12 +40,14 @@ const app = createApp({
     bind(catalogPlugin.externalRoutes, {
       createComponent: scaffolderPlugin.routes.root,
       viewTechDoc: techdocsPlugin.routes.docRoot,
+      selectedTemplateRoute: scaffolderPlugin.routes.selectedTemplate,
```

Comment:
This needs a changeset too.

Reply:
Can we rename these to `createFromTemplate` throughout? I feel the naming here is a bit unfortunate; it doesn't convey the semantic action, and it shouldn't have the word `Route` in it

Reply:
Indeed the naming is bad, I can change it on my side but not sure if I should in the scaffolder plugin...

Reply:
Done the change on the `catalog` side, @freben you think I should change it in `scaffolder` plugin or not?